requirements.txt

```diff
@@ -1,3 +1,211 @@
+adal==1.2.2
```

Comment:
So are these all new requirements that you need for testing? If so can we put it in a different file? That way if a new user wants to just use the checks and doesn't want to run tests they can get away without installing additional requirements.

Reply:
These are the requirements that arose after doing a `pip freeze`, most of them come from `wrapanapi`

Reply:
If they are specific to wrapanpi we don't need to add them explicitly as pip will pull those by default. Let me check it out locally to see how it works.

Reply:
Sure, I can leave it if you'd prefer, in my opinion it is easiest to use `pip freeze` to keep track of requirements.

Reply:
I think it can be left out. Not sure if that's the best way to do it. Can you check with Mike/Justin please?